.github/workflows/ci-win.yml

```diff
@@ -7,9 +7,11 @@ jobs:
     timeout-minutes: 30
     strategy:
       matrix:
-        node-version:  [14.x, 16.x, 18.x]
-        os:
-          - windows-2019
+        node-version: [14.x, 16.x, 18.x]
```

Comment:
we should probably ad 19.x here as well

Reply:
Good point! Let me try adding it,

Reply:
@mhdawson , I have added Node.JS 19.x and updated the PR description.
Should I add Node.JS 19.x for non-Windows platforms in this PR too?

Reply:
@vmoroz yes, we should have it for all platforms. We probably should have done it in Oct when 19.x was cut.

Reply:
OK, I will do it then, while I am editing these files anyway. :)

Reply:
@mhdawson , I have added Node.JS 19x to the MacOS and Ubuntu runners. I also cleaned up the script a bit.